### ISSUE TITLE
Indentation is only allowed with hard tabs or 2 spaces

### ISSUE THREAD

**Issue by eagletmt** [k0kubun/hamlit#26]
Haml allows N-spaces indentation. Is it intentional?
(Also, the exception seems to be telling a wrong information about spaces.)

## Input
```haml
%div
    %div
```

## Expected
```html
<div>
<div></div>
</div>
```

## Actual
```
inconsistent indentation: 2 spaces used for indentation, but the rest of the document was indented using 4 spaces (Hamlit::SyntaxError)
```

**Comment by k0kubun** [comment]
This was not intentional. I didn't know that Haml allows N-spaces indentation.
Fixed in 176f56d.

**Comment by eagletmt** [comment]
:v:

**Comment by eagletmt** [comment]
The commit causes indentation bugs.
https://github.com/eagletmt/faml/commit/a9a979aa6032e22d8e159e4a2778556179ca9f1b#diff-7e8646ecbe18c3c83bf045e3e4bbd551R20

**Comment by k0kubun** [comment]
> The commit causes indentation bugs.

Your specs included many cases I forgot to care about. :fearful: 
Fixed and released as v1.6.4. Thank you for helping me many times :)

**Comment by eagletmt** [comment]
:+1: https://github.com/eagletmt/faml/commit/73b9258ffc126521495a09512ef6e9844ae4cfef
